### ISSUE TITLE
grunt-istanbul Test coverage

### ISSUE THREAD

**Issue by MangeshMp** [meanjs/mean#561]
is there a plan to use grunt-istanbul test coverage for the server side (mocha) coverage.
I recently used it in 0.3 version of my project and it work fine.

It will be a great addition.

**Comment by ilanbiala** [comment]
@MangeshMp if you'd like to create a PR for 0.4 once it is merged into master, that would be great.

**Comment by syymza** [comment]
I have tried it in my project and, indeed,  it does work fine! @MangeshMp let me know if you are going to deal with the PR, otherwise I can do it :)

**Comment by MangeshMp** [comment]
@syymza I created a PR for this issue and also committed the code.
The solution is for 0.4.0 branch done on Grunt .

**Comment by MangeshMp** [comment]
@syymza you can create a PR because I am having some indentation problems.

**Comment by codydaig** [comment]
#569

**Comment by lirantal** [comment]
thanks @codydaig for cleaning this up, we can indeed close and continue discussing in PR #569
